BBj-Plugins/WebKit
# Integrate web component to webkit

In this PR:
- Integration of web-components into webkit
- Avatar Icon as an example implementation

How to test avatar Icon:
- go widgets/Drawer/components/DrawerFooter/DrawerFooter.bbj
- in line number 58, set `#drawerDataModel!.getAvatarUrl()` as `""`
- you will see a cirlcular avatar containing the name of the user.
